contracts/src/lib.rs

```diff
@@ -3,13 +3,26 @@
 
 #[macro_use]
 extern crate alloc;
+use alloc::string::String;
 
 extern crate common;
+use common::bytesrepr::BytesRepr;
 use common::ext::*;
 use common::value::Value;
 
+fn plus_one(x: i32) -> i32 {
+    x + 1
+}
+
+#[no_mangle]
+pub extern "C" fn plus_one_ext() {
+    let x: i32 = get_arg(0);
```

Comment:
How does it differentiate between different methods?

Reply:
`get_arg` is asking the host for whatever arguments the present contract execution was called with. It's kind of like getting arguments from the command line, except all methods have the ambient authority to look at the arguments.
These should really only be used in functions that will become entry points to contracts stored on-chain.